server/commands/hubs.py

```diff
@@ -618,17 +649,46 @@ def ooc_cmd_follow(client, arg):
 
 def ooc_cmd_unfollow(client, arg):
     """
-    Stop following whoever you are following.
-    Usage: /unfollow
+    Stop following whoever you are following, or free someone from being forced to follow.
+    Usage: /unfollow or /unfollow <id>
     """
-    try:
-        client.send_ooc(
-            f"You are no longer following [{client.following.id}] {client.following.showname}."
-        )
-        client.following = None
-    except:
-        client.following = None
-        raise ClientError("You're not following anyone!")
+    if len(arg) == 0:
+        if client.forced_to_follow and not client.is_mod and client not in client.area.area_manager.owners:
```

Comment:
This should check if .following still exists, because the client you're following may disconnect, get deleted, etc.